ci/tests/update-channel.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -euo pipefail
+
+HERMES_BIN=${HERMES_BIN:-${HOME}/build/ibc-rs/release/hermes}
+
+echo "Creating client..."
+"${HERMES_BIN}" tx raw create-client basecoin-0 ibc-0
```

Comment:
Can we also create a client in the opposite direction, namely a client hosted on `ibc-0` for chain `basecoin-0`?

Probably:
```
"${HERMES_BIN}" tx raw create-client ibc-0 basecoin-0
"${HERMES_BIN}" tx raw update-client ibc-0 07-tendermint-0
```

Reply:
I've just added a commit now to create a connection in the opposite direction, but @hu55a1n1 mentioned [here](https://github.com/informalsystems/basecoin-rs/pull/20#issuecomment-963366521) that it may intermittently fail. Does that still hold true @hu55a1n1?

Reply:
Sorry, that comment is outdated. 😅 The connection creation shouldn't fail now. 🤞

Reply:
Awesome, looks like the bidirectional connection integration test passes :grin: :+1: